backend/audit/models.py

```diff
@@ -218,12 +218,10 @@ class STATUS:
         null=True,
     )
 
-    report_id = models.CharField(max_length=17, unique=True)
+    report_id = models.CharField(unique=True)
```

Comment:
I _think_ `unique=True` only works with `max_length` of 255 or less. https://docs.djangoproject.com/en/4.2/ref/databases/#notes-on-specific-fields

Reply:
That's under the MySQL section. This may, or may not, apply to Postgres. (It's not in the Postgres notes. But, if we were to switch DB engines, this could be A Thing.)